### ISSUE TITLE
SEPA Orders Not "Processing" once payment clears

### ISSUE THREAD

**Issue by KennyStier** [woocommerce/woocommerce-gateway-stripe#2376]
**Describe the bug**
After a customer places an order using SEPA, the order status is set to "On hold", which is correct. But even after the payment clears, the Woocommerce order does not change status to "Processing" without manual intervention.

**Expected behavior**
Once SEPA payment is complete, order is set to "Processing" so that it can be fulfilled.

**Screenshots**
![Screenshot from 2022-07-01 15-18-52-obfuscated](https://user-images.githubusercontent.com/4749568/176968964-b0462269-c188-400f-9d3d-0cc5d3b3061b.png)
![Screenshot from 2022-07-01 17-02-59-obfuscated](https://user-images.githubusercontent.com/4749568/176968965-ae8a73b6-ef6d-4a66-ae94-5af3f54d85a4.png)

**Environment (please complete the following information):**
 - WordPress Version: 6.0
 - WooCommerce Version: 6.6.1
 - Stripe Plugin Version: 6.4.3
 - Webserver: Caddy 2, PHP 7.4.8
 - Any other plugins installed: yes

**Comment by ricardo** [comment]
Hi @KennyStier. SEPA payments rely on webhooks to update the order status. Can you please confirm if your webhooks are properly set? See the [Stripe webhooks documentation](https://woocommerce.com/document/stripe/#webhooks) for further details.

If your webhooks are configured correctly, but SEPA orders still don't update, please send us a snippet of your Stripe logs from **WooCommerce > Status > Logs** once the SEPA order is processed.

**Comment by KennyStier** [comment]
Ah, I see. For some reason I was using the wrong webhook secret. Thanks for your assistance!
